app/controllers/computacenter/responsible_body_changes_controller.rb

```diff
@@ -0,0 +1,88 @@
+class Computacenter::ResponsibleBodyChangesController < Computacenter::BaseController
```

Comment:
It'd be nice to add Pundit authorisation to this controller. It's not strictly necessary because `Computacenter::BaseController` already does authorisation, but adding it now makes the job of moving to Pundit eventually easier.

Reply:
Done in 47ca5957